### PR TITLE
Feature/scoresummary

### PR DESCRIPTION
**Pull request by govTechSatish**
Integration with Scoring service & UI of answer page.
Added component for Crum token

### CODE REVIEW

File: app/templates/answers.njk [hunk 1]
```diff
@@ -20,35 +23,98 @@
...
+          </ul>
+            {% endfor %}
+        </th>
+      <td class="govuk-table__cell govuk-!-padding-left-2">
+       <br><br><br>
+        {{ question.rating.band }}
+      </td>
+      <td class="govuk-table__cell">
+        <div class="govuk-!-margin-9">
+        <p>{{question.fundingPriorities}}</p>
```
Comment (paulsimonandrews) [inline app/templates/answers.njk:94]: Change to `<p>{{question.fundingPriorities | safe}}</p>` to make it render the `<br/>`  in `question.fundingPriorities`

File: app/templates/500.njk [hunk 2]
```diff
@@ -7,6 +7,9 @@
         <div class="govuk-grid-column-two-thirds">
           <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
           <p class="govuk-body">Try again later.</p>
+          <p class="govuk-body">We have not saved your answers. When the service is available, you will have to start again.</p>
+          <p>Contact the RPA Helpline if you have any questions.</p>
```
Comment (paulsimonandrews) [inline app/templates/500.njk:11]: add `class="govuk-body"`

File: app/templates/500.njk [hunk 3]
```diff
@@ -7,6 +7,9 @@
...
+          <p>Telephone: 03000 200 301</p>
```
Comment (paulsimonandrews) [inline app/templates/500.njk:12]: Add `class="govuk-body"`

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Two more places to add the gov-uk class

File: app/templates/500.njk (hunk 2 above)
Comment (govTechSatish) [reply app/templates/500.njk:11]: Updated.

**Review by paulsimonandrews — APPROVED** [review]

**Pull request merged**